x/oracle/keeper/request.go

```diff
@@ -61,22 +61,28 @@ func (k Keeper) ProcessExpiredRequests(ctx sdk.Context) {
 	// request. We also resolve requests to status EXPIRED if they are not yet resolved.
 	for ; currentReqID <= lastReqID; currentReqID++ {
 		req := k.MustGetRequest(ctx, currentReqID)
+
 		// This request is not yet expired, so there's nothing to do here. Ditto for
 		// all other requests that come after this. Thus we can just break the loop.
 		if req.RequestHeight+expirationBlockCount > ctx.BlockHeight() {
 			break
 		}
+
```

Comment:
Why we remove this if?

Reply:
At first, I thought we would treat requests as expired after 100 blocks. But seem like it makes sense to bring it back.